src/modelmanager.hpp

```diff
@@ -50,6 +52,10 @@ class CustomNodeLibraryManager;
 class MetricRegistry;
 class ModelConfig;
 class FileSystem;
+#if (MEDIAPIPE_DISABLE == 1)
```

Comment:
You don't need to ifdef forward declares.

Reply:
Fixed.